### PR TITLE
Add LORIS logo to README

### PR DESCRIPTION
**Pull request by johnsaigle**
Screen Shot 2019-12-09 at 15 51 22

_Result of adding the image with width set to 50%_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) [![Documentation Status](https://readthedocs.org/projects/acesloris/badge/?version=latest)](https://acesloris.readthedocs.io/en/latest/?badge=latest)
+![](./loris-logo.svg)
```
Comment (christinerogers) [inline README.md]: Love the change, the logo is currently sized extra.  How about re-sizing it using this syntax below?  (no idea if 80 is the right value, might need tinkering)

```suggestion
![](./loris-logo.svg | width=80)
```

**Comment by johnsaigle** [comment]
@christinerogers how's this?

**Comment by christinerogers** [comment]
Thoughts on reducing it to 25%?   50% is still pretty massive @johnsaigle

**Comment by johnsaigle** [comment]
Screen Shot 2019-12-09 at 15 59 27


This is 25%. I like it better bigger because I like that it's eye-catching.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) [![Documentation Status](https://readthedocs.org/projects/acesloris/badge/?version=latest)](https://acesloris.readthedocs.io/en/latest/?badge=latest)
+<img src="./loris-logo.svg" width="50%">
```
Comment (christinerogers) [inline README.md]: 35% looks better to me in preview mode - not too tiny, you're right. 
```suggestion
<img src="./loris-logo.svg" width="35%">
```

Comment (christinerogers) [reply README.md]: The biggest problem with Readmes is that people don't read them, is what i'm really thinking :)

**Comment by christinerogers** [comment]
@driusan  if you're having a hard day and need an easy merge, consider this one...

File: loris-logo.svg [hunk 3]
```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (driusan) [inline loris-logo.svg]: there's already `htdocs/images/LORIS_logo.svg`, I don't think we should duplicate the image in the repo.

**Pull request merged**